### PR TITLE
Issue 11886

### PR DESCRIPTION
**Pull request by twojtylak**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | mentioned in #11886
| License         | MIT

Add channel contact phone number

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```
Comment (loevgaard) [inline src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php:12]: Remove

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+final class Version20201017150005 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Add contact phone number to channel';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```
Comment (loevgaard) [inline src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php:22]: Remove

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        $this->addSql('ALTER TABLE sylius_channel ADD contact_phone_number VARCHAR(255) DEFAULT NULL');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
```
Comment (loevgaard) [inline src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php:28]: Remove

**Review by loevgaard — APPROVED** [review]
Very nice PR, @twojtylak 🎉

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php (hunk 2 above)
Comment (twojtylak) [reply src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php]: Done

File: src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php (hunk 3 above)
Comment (twojtylak) [reply src/Sylius/Bundle/CoreBundle/Migrations/Version20201017150005.php]: Done

**Pull request closed without merging**